### PR TITLE
Removes a0 realm reference

### PR DESCRIPTION
**Pull request by pohannigansfx**
This was the only reference to the discontinued a0 realm in the Integrations repo (file is heroku/README.md). If you are reviewing this pull request, please ignore any additional commits beyond that file -- merge conflicts in the original pull request have been resolved.

### CODE REVIEW

File: aws-ecs/README.md.jinja [hunk 1]
```diff
@@ -19,7 +18,8 @@
 
 You can use SignalFx to monitor the Amazon EC2 Container Service (ECS) through [Amazon Web Services (AWS)](https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.amazon.web.services.html).
 
-For greater insight into your Amazon ECS environment, the SignalFx Smart Agent can autodiscover services and provide more in-depth metrics about your containers that are running in ECS. See the [installation procedure](#installation-procedure) for more information.
+
```
Comment (brianashby-sfx) [inline aws-ecs/README.md.jinja:21]: extra line break?

File: couchbase/README.md.jinja [hunk 2]
```diff
@@ -33,18 +33,15 @@
...
 ### Smart Agent Monitor
 
 This is a Smart Agent monitor for [Couchbase](https://www.couchbase.com/)
-<<<<<<< HEAD:couchbase/README.md.jinja
-that uses the [couchbase collectd Python plugin](https://github.com/signalfx/collectd-couchbase) that you configure to collect metrics from Couchbase server instances.
-=======
+
 that uses the [Couchbase collectd Python plugin](https://github.com/signalfx/collectd-couchbase) that you configure to collect metrics from Couchbase server instances.
->>>>>>> dcb68c5fffabac207e0a84b514a416e0161627e7:couchbase/SMART_AGENT_MONITOR.md
+
```
Comment (brianashby-sfx) [inline couchbase/README.md.jinja:44]: extra line break?

File: couchbase/README.md.jinja [hunk 3]
```diff
@@ -77,8 +73,13 @@ db.createUser( {
...
 
-The [`data_monitoring` role](https://docs.couchbase.com/server/6.6/learn/security/roles.html#data-monitor) has the minimum permissions required to use the integration. For
-more information on the associations of privileges to resources and roles, see [Roles.](https://docs.couchbase.com/server/6.6/learn/security/roles.html)
+```
+db.createUser( {
+  user: "collectd",
+  pwd: "collectd",
+  roles: [ { role: "readAnyDatabase", db: "admin" }, { role: "clusterMonitor", db: "admin" } ]
+});
+```
```
Comment (brianashby-sfx) [inline couchbase/README.md.jinja:82]: what's the context for this? the preceding line doesn't indicate exactly what this is.

File: mysql/README.md.jinja [hunk 4]
```diff
@@ -1,7 +1,7 @@
-<<<<<<< HEAD:mysql/README.md.jinja
 # ![](./img/integrations_mysql.png) MySQL
 {% import "macros.jinja" as macros %}
 
+
```
Comment (brianashby-sfx) [inline mysql/README.md.jinja:4]: extra line break?

File: mysql/README.md.jinja [hunk 5]
```diff
@@ -13,19 +13,6 @@
...
-{% if target == "docs" -%}
-# MySQL
-
-- [Overview](#overview)
-- [Setup](#setup)
-- [Built-in Content](#built-in-content)
-- [Usage](#usage)
-- [Metrics](#metrics)
-
->>>>>>> dcb68c5fffabac207e0a84b514a416e0161627e7:mysql/SMART_AGENT_MONITOR.md
```
Comment (brianashby-sfx) [inline mysql/README.md.jinja:28]: was this deleted  to address a merge conflict with master?

File: mysql/README.md.jinja [hunk 6]
```diff
@@ -61,21 +47,18 @@ db.createUser( {
 });
 ```
 
-=======
-To create a MySQL user for collectd:
+
```
Comment (brianashby-sfx) [inline mysql/README.md.jinja:50]: extra line break?

File: mysql/README.md.jinja [hunk 7]
```diff
@@ -61,21 +47,18 @@ db.createUser( {
 });
 ```
 
-=======
-To create a MySQL user for collectd:
```
Comment (brianashby-sfx) [inline mysql/README.md.jinja:65]: why was this deleted?

File: mysql/README.md.jinja [hunk 8]
```diff
@@ -61,21 +47,18 @@ db.createUser( {
...
- -- Give appropriate permissions
- -- ("GRANT USAGE" is synonymous to "no privileges")
- GRANT USAGE ON *.* TO 'collectd'@'localhost';
- -- Permissions for the stats options
- GRANT REPLICATION CLIENT ON *.* TO 'collectd'@'localhost';
+db.createUser( {
+  user: "collectd",
+  pwd: "collectd",
+  roles: [ { role: "readAnyDatabase", db: "admin" }, { role: "clusterMonitor", db: "admin" } ]
+});
```
Comment (brianashby-sfx) [inline mysql/README.md.jinja:57]: is this to address a conflict with master? if so, what's the context for lines 53-57?

**Review by brianashby-sfx — CHANGES_REQUESTED** [review]

File: mysql/README.md.jinja (hunk 5 above)
Comment (pohannigansfx) [reply mysql/README.md.jinja:28]: yes

File: aws-ecs/README.md.jinja (hunk 1 above)
Comment (pohannigansfx) [reply aws-ecs/README.md.jinja:21]: it was

File: mysql/README.md.jinja (hunk 7 above)
Comment (pohannigansfx) [reply mysql/README.md.jinja:65]: inadvertently

File: mysql/README.md.jinja (hunk 8 above)
Comment (pohannigansfx) [reply mysql/README.md.jinja:57]: Green task creates MySQL user for collectd, but that wasn't apparent with intro line gone (since restored)

**Comment by pohannigansfx** [comment]
Removed 3 of the 4 files (from the commit) as unrelated to realm update.

**Comment by pohannigansfx** [comment]
Resolved file conflicts and incorporated review suggestions. This includes minor updates in addition to the realm fix because pulling jinja readme files from the request created other issues.
